test/battle/move_effect/pledge.c

```diff
@@ -76,27 +76,6 @@ DOUBLE_BATTLE_TEST("Rainbow flinch chance does not stack with Serene Grace")
     }
 }
 
-DOUBLE_BATTLE_TEST("Rainbow flinch chance does not stack with Serene Grace if mvoe Triple Arrows is used")
```

Comment:
Why was this removed?

Reply:
There's no need for this move to have a separate test - it uses the same flinch move effect as the test above.